SmartDeviceLink/SDLViewControllerPresentable.h

```diff
@@ -8,15 +8,24 @@
 
 #import <UIKit/UIKit.h>
 
-/**
- *  A protocol used to tell a view controller to present another view controller. This makes testing of modal VCs' presentation easier.
- */
+NS_ASSUME_NONNULL_BEGIN
+
+/// A protocol used to tell a view controller to present another view controller. This makes testing of modal VCs' presentation easier.
 @protocol SDLViewControllerPresentable <NSObject>
 
-@property (strong, nonatomic) UIViewController *lockViewController;
-@property (assign, nonatomic, readonly) BOOL presented;
+/// The view controller to be presented as a lockscreen
+@property (strong, nonatomic, nullable) UIViewController *lockViewController;
+
+/// Whether or not the lockscreen should be presented
+@property (assign, nonatomic, readonly) BOOL showLockScreen;
```

Comment:
This would need to be updated if the above is updated.

Reply:
Fixed.